--- hardinfo-master/modules/computer.c~	2020-02-10 17:36:35.000000000 +0100
+++ hardinfo-master/modules/computer.c	2020-04-27 09:11:36.756595011 +0200
@@ -544,7 +544,7 @@
     info_set_view_type(info, SHELL_VIEW_DETAIL);
 
     info_add_group(info, _("Computer"),
-        info_field_printf(_("Processor"),
+        info_field_printf(_("Processor"), "%s",
             idle_free(module_call_method("devices::getProcessorNameAndDesc"))),
         info_field_update(_("Memory"), 1000),
         info_field_printf(_("Machine Type"), "%s",
