diff -ruN hardinfo-0.4.2.3./configure hardinfo-0.4.2.3/configure
--- hardinfo-0.4.2.3./configure	2007-11-04 14:14:47.000000000 +0100
+++ hardinfo-0.4.2.3/configure	2007-11-21 20:03:42.654506493 +0100
@@ -157,7 +157,7 @@
 	case $? in
 		0)
 			SOUP_FLAGS=`pkg-config libsoup-2.2 --cflags --static`
-			SOUP_LIBS=`pkg-config libsoup-2.2 --libs --static`
+			SOUP_LIBS="`pkg-config libsoup-2.2 --libs --static` -lgthread-2.0"
 			echo "found `pkg-config libsoup-2.2 --modversion`"
 			SOUP=1
 			break ;;
