--- hardinfo/hardinfo/pci_util.c~	2020-04-27 08:59:19.000000000 +0200
+++ hardinfo/hardinfo/pci_util.c	2020-04-27 09:06:32.013271410 +0200
@@ -46,6 +46,7 @@
         }
     }
     char *file_search_order[] = {
+        g_strdup("/lib/hwdata/pci.ids"),
         g_strdup("/usr/share/hwdata/pci.ids"),
         g_strdup("/usr/share/misc/pci.ids"),
         g_build_filename(g_get_user_config_dir(), "hardinfo", "pci.ids", NULL),
--- hardinfo/modules/devices.c~	2020-04-27 08:59:19.000000000 +0200
+++ hardinfo/modules/devices.c	2020-04-27 09:08:10.833268282 +0200
@@ -830,7 +830,9 @@
 
 void hi_module_init(void)
 {
-    if (!g_file_test("/usr/share/misc/pci.ids", G_FILE_TEST_EXISTS)) {
+    // probably they don't have SOAP entry for gz compressed action
+    // but URL exists for that: http://pci-ids.ucw.cz/v2.2/pci.ids.gz
+    if (!g_file_test("/lib/hwdata/pci.ids", G_FILE_TEST_EXISTS)) {
         static SyncEntry se = {
              .fancy_name = N_("Update PCI ID listing"),
              .name = "GetPCIIds",
